playwright/fixtures/index.ts

```diff
@@ -308,6 +314,13 @@ export const defaultTest = (
       // Propagate the modified page to the test
       await use(page);
     },
+
+    /**
+     * Mocks
+     */
+    procedures: async ({ page }, use) => {
+      await use(await MockManagerProcedures.init(page));
```

Comment:
Here this is broken in typescript with any :/

 I guess it's because of the confusion with class for typescript.
 
 Maybe we should create a classic TS function to handle this? It could remove the problem with the constructor. WDYT?

Reply:
It might be because I renamed `mockManagerProcedures.ts` to `MockManagerProcedures.ts` and eslint/tsserver isn't picking it up. I have full typing on my end.

Git properly shows the rename that was done via `git mv`. If it's still giving you trouble, maybe we can just rename the file something else temporarily.

A classic function is possible, but seems a bit messy considering we already have a class that encapsulates the mocking logic. The alternative was:

```ts
const mockManagerProcedures = new MockManagerProcedures()
await mockManagerProcedures.setup(page)
```

The `setup`/`init` methods are necessary because constructors cannot return Promises, thus they cannot use `async`/`await`.

Reply:
Another thing to consider re: using a plain function:

We may want to check the list of mocked procedures. Returning the full `MockProceduresClass` allows us to read the instance's `procedures` property. Providing a single `mockProcedures` fixture wouldn't allow that, unless we returned an object.